RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Live.swift

```diff
@@ -152,8 +152,6 @@ extension CloudBackupClient {
 					}
 
 					let savedRecord = try await uploadProfileSnapshotToICloud(profileSnapshot, id: profile.id, existingRecord: backedUpRecord)
-					// Migration completed, deleting old copy
```

Comment:
@kugel3 Before merging this, I want to actually understand the reasoning behind deleting the profile from the secure storage.

IMO, the Wallet should be able to locate the profile in secure storage even after the "backup migration". To me, it is wrong to assume that backup migration means uploading the profile from the Keychain to the cloud container and then deleting it from the Keychain. 

The SecureStorage(KeyChain) is the actual local storage of the Profile, and the profile should be suddenly gone. A more correct interpretation of migration maybe is:
- Upload the profile from keychain to cloud.
- Disable keychain sync for the said profile.

Probably I am missing something as you have worked a lot more on this feature.

Reply:
I didn't see this comment before, but it was decided last week to not actually delete non-current profiles when they are "migrated". Matt still wants to delete the currently used profile from backup, but it just hit me that I am not sure we even can do that.

Reply:
Keep in mind that this code only applies to non-current profiles.

Reply:
Interesting that you mention non-current profiles. In the context of SelectBackup, aren't all profiles non-current? I don't see the migration function to be called from anywhere else than from SelectBackup.

I guess what we have to do to remove the backup from cloud, is to disable iCloud sync of the profile in SecureStorage, which we should do anyway.